specification/storagesync/resource-manager/Microsoft.StorageSync/stable/2018-07-01/storagesync.json

```diff
@@ -2508,15 +2682,211 @@
         }
       }
     },
-    "ServerEndpointUpdateParameters": {
+    "RecallActionParameters": {
       "properties": {
+        "pattern": {
+          "type": "string",
+          "description": "Pattern of the files."
+        },
+        "recallPath": {
+          "type": "string",
+          "description": "Recall path."
+        }
+      },
+      "description": "The parameters used when calling recall action on server endpoint."
+    },
+    "StorageSyncServiceCreateParameters": {
+    "properties": {
+        "location": {
+            "type": "string",
+            "description": "Required. Gets or sets the location of the resource. This will be one of the supported and registered Azure Geo Regions (e.g. West US, East US, Southeast Asia, etc.). The geo region of a resource cannot be changed once it is created, but if an identical geo region is specified on update, the request will succeed."
+        },
         "tags": {
-          "type": "object",
-          "additionalProperties": {
-            "type": "string"
-          },
-          "description": "The user-specified tags associated with the server endpoint."
+            "type": "object",
+            "additionalProperties": {
+                "type": "string"
+            },
+            "description": "Gets or sets a list of key value pairs that describe the resource. These tags can be used for viewing and grouping this resource (across resource groups). A maximum of 15 tags can be provided for a resource. Each tag must have a key with a length no greater than 128 characters and a value with a length no greater than 256 characters."
         },
+        "properties": {}
+    },
+    "required": [
+        "location"
+    ],
+    "description": "The parameters used when creating a storage sync service."
+},
+    "SyncGroupCreateParameters": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ProxyResource"
+        }
+	  ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SyncGroupCreateParametersProperties",
+          "description": "The parameters used to create the sync group"
+        }
+      },
+      "description": "The parameters used when creating a sync group."
+    },
+    "SyncGroupCreateParametersProperties": {
```

Comment:
Why is the property section empty? Are there no properties for this resource?

Reply:
Yes, No properties so far